### DISCUSSION TITLE
You can't break vines on a portal

### DISCUSSION THREAD

**Discussion by LoganDark** [iPortalTeam/ImmersivePortalsMod#1603] [Non-Compatibility Issue Report]
this:
![image](https://user-images.githubusercontent.com/4723091/79850905-69f27400-8379-11ea-8374-d95ad599cb2c.png)

but not this:
![image](https://user-images.githubusercontent.com/4723091/79850916-6f4fbe80-8379-11ea-9134-9e89adec4a62.png)

**Comment by LoganDark** [comment]
you can't even interact with a door on a portal
![image](https://user-images.githubusercontent.com/4723091/79947072-20ac2e00-8426-11ea-8472-970908f0e0e4.png)

**Comment by PoolloverNathan** [comment]
Noticed the door problem with my "it gets bigger on the inside" base
